### ISSUE TITLE
Microsoft.Extensions.Logging.ILoggingBuilder

### ISSUE THREAD

**Issue by ilivecoding** [lovedotnet/UnitOfWork#151]
![图片](https://user-images.githubusercontent.com/7898334/123394981-499ff400-d5d2-11eb-8ad5-202483fc1925.png)

**Comment by onyor** [comment]
did you solve this problem?

**Comment by nappinglady** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by mendozagit** [comment]
Onyor, in my personal opinion this repo unfortunately isn't more maintained 😢

**Comment by onyor** [comment]
I just wanted to know how some methods in the project work.  I was going to do some testing. 

Can't i run this project?

**Comment by mendozagit** [comment]
As far I know, this repo (nuget) doesn't work with higher version of .Net core 3.1

If you are trying to run on .NET5+ probably won't works, or maybe happens something unexpected behavior.

**Comment by onyor** [comment]
Yes.  I'm trying to run it in .NET5  this could be the problem.  Thanks for information. Again I will try with .Net3.1
